test/utils.py

```diff
@@ -127,15 +127,21 @@ def verify_testrunner(testrunner, ts0, ts1='', ts2='', ts3='', tr=''):
 
 def expect_testrunner_pass(logfile_path):
     with open(logfile_path) as file, mmap.mmap(file.fileno(), 0, access=mmap.ACCESS_READ) as log:
-        return re.search(br'INFO:  \[.*\]\[testrunner\]: PASSED (. of . suites passing) \[$SVUnitVersion\]', log)
+        if re.search(br'INFO:  \[.*\]\[testrunner\]: PASSED \(. of . suites passing\) \[.*\]', log):
```

Comment:
I'm maybe missing something, but why you've replaced `$SVUnitVersion` with `.*` regex syntax here?

Commit message does not say anything about it. Maybe there was another other bug that you've fixed here too?

Please elaborate more about this change.

Reply:
Yes, this is bug fix. In log message we have version number of SVUnit in this place e.g.: "SVUnit v3.34.0" but test check if there is "$SVUnitVersio" string not proper value of version. In test env we don't have access to version number so I use ".*" instead
But yes, maybe this should be elaborate in commit log
I will add this, thx

Reply:
Good enough for me, +1.